internal/pkg/remote/endpoint/client.go

```diff
@@ -95,32 +95,32 @@ func (ep *Config) KeyserverClientOpts(uri string, op KeyserverOp) ([]keyclient.O
 	return co, nil
 }
 
-func (ep *Config) LibraryClientConfig(uri string) (*libclient.Config, error) {
+func (ep *Config) RegistryClientConfig(uri string) (*registryclient.Config, error) {
```

Comment:
These should be removed instead of renamed

Reply:
deferred to #71 and #57